apluslms-yamlidator/apluslms_yamlidator/document.py

```diff
@@ -38,7 +42,6 @@ def find_ml(current, keys):
             current = current.setdefault(key, {})
     return current, keys[-1]
 
-
```

Comment:
I prefer to use two blank lines to separate root level functions etc., but often three between different classes.

> Surround top-level function and class definitions with two blank lines.
https://www.python.org/dev/peps/pep-0008/#blank-lines